app/controllers/services_controller.rb

```diff
@@ -0,0 +1,6 @@
+class ServicesController < ApplicationController
+  def register
+    @service = Service.create
+    render json: @service, include: :models
```

Comment:
One service trying to 'claim' responsibility for a model already owned by a service should be a 400-ish error. That's an invalid request.